upgrade/php/ps_1770_update_charset.php

```diff
@@ -57,26 +62,26 @@ function ps_1770_update_charset()
 
     if ($adminFilterTableExists) {
         if ($adminFilterFilterIdExists) {
-            $result &= Db::getInstance()->execute(
+            $result &= DbWrapper::execute(
```

Comment:
To keep $result as a boolean, the check should be

```suggestion
            $result = $result && DbWrapper::execute(
```

Otherwise, PHP makes a [bitwise operation](assertQuerySuccess) which lead to an integer